src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -182,9 +182,11 @@ function BaseTextInput(
             }
 
             const layout = event.nativeEvent.layout;
+            // We need to increase the height for single line inputs to escape cursor jumping on ios
```

Comment:
Small change, you can probably just commit the suggestion. Our style guides usually mention adding a blank line before a comment 
```suggestion

            // We need to increase the height for single line inputs to escape cursor jumping on ios
```